src/grandorgue/config/GOMidiDeviceConfigList.cpp

```diff
@@ -1,113 +1,92 @@
 /*
-* Copyright 2006 Milan Digital Audio LLC
-* Copyright 2009-2021 GrandOrgue contributors (see AUTHORS)
-* License GPL-2.0 or later (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
-*/
+ * Copyright 2006 Milan Digital Audio LLC
+ * Copyright 2009-2021 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
 
 #include "GOMidiDeviceConfigList.h"
 
 #include "config/GOConfigReader.h"
 #include "config/GOConfigWriter.h"
-
 #include "go_limits.h"
 
-GOMidiDeviceConfig* GOMidiDeviceConfigList::FindByLogicalName(
-  const wxString& logicalName
-) const
-{
-  GOMidiDeviceConfig* res = NULL;
+GOMidiDeviceConfig *
+GOMidiDeviceConfigList::FindByLogicalName(const wxString &logicalName) const {
```

Comment:
Please add ``PenaltyReturnTypeOnItsOwnLine: 1000`` for avoiding this kind of break

Reply:
Done